docs/networking/clusternetwork.md

```diff
@@ -31,14 +31,31 @@ Specifications including network devices of the Harvester hosts can be different
 
 Network configuration only works under a certain cluster network. Each network configuration corresponds to a set of hosts with uniform network specifications. Therefore, multiple network configurations are required for a cluster network on non-uniform hosts.
 
-### Network
+### VM Network
 
-A network is an interface in a virtual machine that connects to the host network. As with network configuration, every network except the built-in [management network](./harvester-network.md#management-network) must be under a cluster network.
+A VM network is an interface in a virtual machine that connects to the host network. As with network configuration, every network except the built-in [management network](./harvester-network.md#management-network) must be under a cluster network.
 
 Harvester supports adding multiple networks to one VM. If a network's cluster network is not enabled on some hosts, the VM that owns this network will not be scheduled to those hosts.
 
 Please refer to [network part](./harvester-network.md) for more details about networks.
 
+### Relationship Between Cluster Network, Network Config, VM Network
+The following diagram shows the relationship between cluster network, network config, and VM network.
+
+![](/img/v1.2/networking/relation.png)
+
+All `Network Configs` and `VM Networks` are grouped under a cluster network. To categorize hosts based on their network specifications, a label can be assigned to each host and a network config can be added for each group of hosts using a node selector. E.g., In the diagram above,  for `ClusterNetwork-A` i, hosts are divided into three groups as follows:
+   - The first group includes host0 and host2, which correspond to `network-config-A`.
+   - The second group includes host1, which corresponds to `network-config-B`.
+   - The third group includes the remaining hosts (host3, host4, and host5), which do not have any related network config and therefore do not belong to `ClusterNetwork-A`.
+
+The cluster network is only effective on hosts that are covered by the network configuration. If a VM uses a `VM network` under a specific cluster network, it can only be scheduled on a host where the cluster network is active. In the diagram, we can see that:
+   - `ClusterNetwork-A` is active on host0, host1, and host2. `VM0` uses `VM-network-A`, so it can be scheduled on any of these hosts.
```

Comment:
```suggestion
- `ClusterNetwork-A` is active on host0, host1, and host2. `VM0` uses `VM-network-A`, so it can be scheduled on any of these hosts.
```